tests/images.spec.ts

```diff
@@ -18,3 +24,43 @@ test("search for custom image and create an instance from it", async ({
 
   await deleteInstance(page, customInstance);
 });
+
+test("Export an image", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  const imageAlias = await createImageFromInstance(page, instance);
+
+  await page.getByRole("link", { name: "Images", exact: true }).click();
+  await page.getByPlaceholder("Search").click();
+  await page.getByPlaceholder("Search").fill(imageAlias);
+
+  await expect(page.getByText(imageAlias)).toBeVisible();
+  const imageName = await getImageNameFromAlias(page, imageAlias);
+  await page.getByLabel("export image").click();
+  await page.waitForSelector(
+    `text=Image ${imageName} was successfully downloaded. Please check your downloads folder.`,
+  );
+
+  await deleteImage(page, imageName);
+  await deleteInstance(page, instance);
+});
+
+test("Upload an image", async ({ page }) => {
+  const IMAGE_FILE = "./tests/fixtures/e2e-test-image.tar.gz";
+  const imageAlias = `${randomInstanceName()}-1`;
```

Comment:
```suggestion
  const imageAlias = `${randomImageName()}-1`;
```
I think we should create a function creating us random image aliases in the image helper. We can also use it in `createImageFromInstance`